tools/doc/node_modules/marked/LICENSE

```diff
@@ -1,19 +0,0 @@
-Copyright (c) 2011-2014, Christopher Jeffrey (https://github.com/chjj/)
```

Comment:
I think this means we'll have to fix the license-builder script:

https://github.com/nodejs/node/blob/7dca3297bce9c19a0451be1b130c8fcab564fe04/tools/license-builder.sh#L74-L75

Reply:
Updated license-builder.sh. Thanks.